irmaclient/session.go

```diff
@@ -660,13 +668,18 @@ func panicToError(e interface{}) *irma.SessionError {
 // background jobs. This function is idempotent, doing nothing when called a second time. It
 // returns whether or not it did something.
 func (session *session) finish(delete bool) bool {
-	if !session.done {
-		if delete && session.IsInteractive() {
-			session.transport.Delete()
-		}
-		session.client.nonrevRepopulateCaches(session.request)
-		session.client.sessions.remove(session.token)
-		session.done = true
+	irma.Logger.Warn(fmt.Sprintf("Deleting session %p", session))
```

Comment:
- Instead of saying `Logger.Warn(fmt.Sprintf`, you can just say `Logger.Warnf` here
- Better would be to put it in a field: `irma.Logger.WithField("session", session).Warn("...")`
- Please don't use the `Warn` level for non-exceptional situations like these

Reply:
Woeps, have removed the left-over debugging code